.travis.yml

```diff
@@ -8,6 +8,9 @@ install:
 script:
     - make all-tests
 
+after_success:
+    - .scripts/travis/push-image.sh search
```

Comment:
@giorgiosironi Will this push the image that was created for tests? If so, I wouldn't suggest pushing this image. We can start to look at creating a production image and tests using the production image for CI?